canonical/canonical.com
# WD-17111 - Implement new data/streaming page and form

## Done

Implement new page: `/data/streaming`

## QA

- Navigate to /data/streaming
- Ensure the design is aligned with the Figma file. 
- Ensure the page content aligns with the copy doc

## Issue / Card

- [WD-17111](https://warthogs.atlassian.net/browse/WD-17111)
